.changeset/tidy-otters-repair.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/ui5-info': minor
```

Comment:
I believe that a `patch` update is sufficient. Please change to patch

Reply:
ok

Reply:
done